frontend/components/Pages/HelpAndSupportPage.tsx

```diff
@@ -45,6 +62,14 @@ export const HelpAndSupport = () => {
         <a target="_blank" href={FAQ_URL} className="mb-8">
           Read FAQ {UNICODE_SYMBOLS.EXTERNAL_LINK}
         </a>
+        <a
```

Comment:
can we please check latestTag for null before rendering this? the link might be broken while the tag is loading